addons/website/static/src/scss/configurator.scss

```diff
@@ -84,14 +84,46 @@
                 min-width: 13ch;
             }
 
-            .industry_selection.o_step_todo {
-                min-width: 17ch;
-            }
-
             .o_configurator_purpose_dd.o_step_todo {
                 min-width: 12ch;
             }
 
+            .o_configurator_industry_wrapper {
+                position: relative;
+                display: inline-block;
+                height: $line-height-base * 1em;
+                min-width: 17ch;
+                cursor: pointer;
+
+                input {
+                    position: absolute;
+                    width: 100%;
+                    height: $line-height-base * 1em;
+                    padding: 0;
+                    border: none;
+                    border-bottom: 3px solid theme-color(primary);
+                    color: theme-color(primary);
+                    font-weight: 500;
+                    font-style: italic;
+                    text-indent: 5px;
```

Comment:
Share the code please. Everytime you use CTRL-C instead of CTRL-X, a kitten dies :wink: :grimacing: 
`&, input { ... }`